common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -216,6 +219,7 @@ public void run() {
                             }
                             finalFuture.setCleanedUp();
                         }
+                        //DiagnosticContext.clear();
```

Comment:
Did we mean to do this or not? If not, then let's remove the comment.

Reply:
38b8d704207fb078d883d644a73e0835fa8ded39